libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -1,7 +1,14 @@
-import { getConfig, getMetadata, loadStyle, loadLana } from '../../../utils/utils.js';
+import { getConfig, getMetadata, loadStyle, loadLana, decorateLinks } from '../../../utils/utils.js';
 import { processTrackingLabels } from '../../../martech/attributes.js';
+import { replaceText } from '../../../features/placeholders.js';
 
 loadLana();
+const allowedOrigins = [
+  'https://www.adobe.com',
+  'https://business.adobe.com',
+  'https://blog.adobe.com',
+  'https://milo.adobe.com',
+];
```

Comment:
Should this be defined in the config level instead?  Possibly hard code these in milo but allow consumers to add their site to the list?

Reply:
Yes, that could make sense, I added a TODO to revisit this when porting this closer to milo core and out of the GNAV.